src/reference/config/file-format.rst

```diff
@@ -178,3 +179,7 @@ following way:
    We only use one set of square brackets at each level when writing nested
    sections on one line like this. But in the file, each sub-section
    gets additional square brackets as shown above.
+
+.. note::
+   Validation warnings also enclose the final item name in brackets to avoid
+   ambiguity when the name includes spaces.
```

Comment:
Actually I think this is done inconsistently, and should be fixed. I'll DRAFT this PR until that's resolved.

Reply:
(Removed this)